nengo_gui/components/htmlview.py

```diff
@@ -10,31 +12,50 @@ class HTMLView(Component):
 
     def __init__(self, obj):
         super(HTMLView, self).__init__()
-        self.obj = obj
-        self.obj_output = obj.output
         self.data = collections.deque()
 
+        self.obj = obj
+        self.obj_output = self.obj.output
```

Comment:
So ``self.obj_output`` always represents the original node function. If ``not callable_html``, then we need to keep this around since we overwrite the node output, and need to be able to set it back in ``remove_nengo_objects``. It's not random which is used where: ``self.obj.output`` is only used when we're setting or re-setting the node output, and ``self.obj_output`` is used whenever we want to call the old node function. This is consistent with how this class currently works in ``master``.